app/User.php

```diff
@@ -1,11 +1,14 @@
 <?php
+
 namespace App;
 use Illuminate\Notifications\Notifiable;
 use Illuminate\Contracts\Auth\MustVerifyEmail;
 use Illuminate\Foundation\Auth\User as Authenticatable;
 class User extends Authenticatable
+
```

Comment:
There should be no space here. it should be before the class

Reply:
I made a mistake.
The code has been fixed.